PhysicsTools/PatAlgos/plugins/CaloJetSlimmer.cc

```diff
@@ -0,0 +1,63 @@
+
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "DataFormats/Common/interface/RefVector.h"
+
+#include "CommonTools/UtilAlgos/interface/StringCutObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/SingleObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/ObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/SingleElementCollectionSelector.h"
+
+#include "DataFormats/JetReco/interface/CaloJet.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
```

Comment:
hi @jingyuluo - why is this dependency needed? 

Depending on the answer to that question, perhaps RecoJets will be a better location for this code.